config/policies.js

```diff
@@ -40,6 +40,7 @@ let policyObject = {
     'get-players-datatable': ['isLoggedIn', 'roles/managePlayers'],
     'server-info-view': ['isLoggedIn', 'roles/hasAccess'],
     'check-donator': 'isLoggedIn',
+    'server-tile': ['isLoggedIn'],
```

Comment:
I think this should be locked down a bit better, right?

`roles/viewDashboard` OR `roles/useTracking` probably? WDYT?

Reply:
`roles/hasAccess` is also an option, the pages themselves are locked down by the policies. This could allow someone with access to CSMM but not access to dashboard to load map tiles via API though.

Reply:
so i'm super scared by this in general, since its going to get 60 page hits when its loaded. Does it also update? thats another 60 per time period.

Are they at all expensive?

Reply:
Yep I see what you're saying. They're not super expensive but some queries need to happen. Possibly we can try running a benchmark with Autocannon (or something) to see how heavy it is on the host?

Another possibility is to create a new permission field "canSeeMap" with a separate policy that's designed to be as light weight as possible?